data/sql_setup/prepare_schedule_a.sql

```diff
@@ -96,7 +96,14 @@ begin
         -- visit here:
         -- https://www.postgresql.org/docs/current/static/plpgsql-statements.html#PLPGSQL-STATEMENTS-DIAGNOSTICS
         if FOUND then
-            two_year_transaction_period = get_transaction_year(new.contb_receipt_dt, view_row.rpt_yr);
+            -- Requires that the hstore extension is installed.
+            if hstore(new) ? 'contb_receipt_dt' then
+                contb_receipt_dt = new.contb_receipt_dt;
+            else
+                contb_receipt_dt = cast(null as timestamp);
```

Comment:
Actually this may not be an issue - the view pulls `rpt_yr` from the underlying `f_item` table anyway, so passing `null` in for the first argument will just force it to use that value regardless.  This should be fine then and keeps things simple. :-)